tests/run-build.sh

```diff
@@ -8,6 +8,8 @@ if [ -f /opt/change-go-version.sh ]; then
     change-go-version 1.4
 fi
 
+SRC_CHANGES=$(git diff-tree --no-commit-id --name-only -r HEAD..origin/master | grep -vE "^Documentation/" | wc -l)
```

Comment:
Fair enough.

`HEAD..origin/master` might be Semaphore specific and developers might name their git-remotes differently. Do you think we should check `$SEMAPHORE=true` and `$CI=true`? See [Semaphore environment variables](https://semaphoreci.com/docs/available-environment-variables.html)